origins_translations/js/origins-translations.js

```diff
@@ -23,31 +63,33 @@
       }
 
       // Lookup the translation for the UI title.
-      $.ajax({
-        url: '/origins-translations/translation-link-ui/title/' + lang_code,
-      })
-        .done(function(data) {
-          if (data) {
-            $(elm).val(data);
-          }
-        });
+      $.getJSON({
+        url: '/origins-translations/translation-ui/languages',
+      }).done(function(data) {
+        if (data) {
+          $button.text(data[lang_code][3]);
```

Comment:
Is this value always present when called? Could be useful to set some fallbacks, in case the other end returns something unexpected in the future.

Reply:
"this value" meaning data[langcode][3]